R/verb-pivot-longer.R

```diff
@@ -169,6 +189,26 @@ get_measure_column_exprs <- function(name, value, values_transform) {
   )
 }
 
+apply_name_repair_pivot_longer <- function(id_cols, spec, names_repair) {
+  nms_map_df <- vctrs::vec_rbind(
```

Comment:
Would you mind adding a brief comment describing what's happening here?

Reply:
Done